packages/server/src/runtime/proxy.rs

```diff
@@ -82,6 +82,11 @@ impl tg::Handle for Proxy {
 		// Replace the path with the host path.
 		if let Some(path) = &mut arg.path {
 			*path = self.host_path_for_guest_path(path.clone())?;
+		} else {
+			// If there's no path set (internal checkout) and the VFS is enabled, ignore the request.
+			if self.server.options.vfs {
+				return Ok(tg::artifact::checkout::Output::default());
```

Comment:
I don't think we want to return the empty path here, but the equivalent of `/.tangram/artifacts/<id>`